control-plane/csi-driver/src/context.rs

```diff
@@ -52,6 +52,10 @@ pub enum Parameters {
     CloneFsIdAsVolumeId,
     #[strum(serialize = "fsId")]
     FsId,
+    #[strum(serialize = "poolTopologyAffinity")]
```

Comment:
Do we need topology in the name?
ex: poolAffinity vs poolTopologyAffinity

Reply:
It makes things look consistent like `poolTopologyAffinity` , `poolTopologySpread`, `nodeTopologyAffinity`, `nodeTopologySpread`

Reply:
Yes, good point